### DISCUSSION TITLE
No suggestion button in translation screen?

### DISCUSSION THREAD

**Discussion by jmctune** [WeblateOrg/weblate#9611] [Q&A] [answered]
Hello,

I've created a new team named "Suggestors" and have only given them the "Add Suggestion" and "Access repository" roles. I've assigned a test user to this team. When I log into the team, I am not seeing the "Suggest" button in the translation. Did I do something incorrectly?

![image](https://github.com/WeblateOrg/weblate/assets/17505625/554bf047-636c-42b9-a35e-40aa37775df9)

![image](https://github.com/WeblateOrg/weblate/assets/17505625/0df60a37-007c-4e02-9f98-8623028506e3)

**Comment by nijel** [comment, marked as answer]
Are suggestions enabled on the component? See https://docs.weblate.org/en/latest/admin/projects.html#enable-suggestions

**Reply by jmctune** [reply]
Thanks - that was definitely it.
